src/Storages/System/StorageSystemStackTrace.cpp

```diff
@@ -324,7 +324,7 @@ Pipe StorageSystemStackTrace::read(
             sigval sig_value{};
 
             sig_value.sival_int = sequence_num.load(std::memory_order_acquire);
-            if (0 != ::sigqueue(tid, sig, sig_value))
+            if (0 != ::sigqueue(static_cast<int>(tid), sig, sig_value))
```

Comment:
Well since there are other platforms I prefer to keep it `int` instead of `pid_t` to avoid extra failures. Objections?
P.S. at least `pid_t` not `__pid_t`

Reply:
Yepp, it is okay to keep `int'.